README.md

```diff
@@ -1,49 +1,52 @@
+The Exchange Union Daemon ([`xud`](https://github.com/ExchangeUnion/xud)) is the reference node implementation powering [OpenDEX](https://opendex.network), a decentralized exchange (DEX) built on top of the [Lightning](https://lightning.network/) and [Raiden](https://raiden.network/) network. `xud` brings individual traders, market makers and exchanges onto OpenDEX to form a single global trading network.
 
-`xud` brings individuals and exchanges onto the same network to form a single global liquidity pool. This gives users a choice to either trade directly on the DEX by running `xud`, managing private keys and software stack, *or* to conveniently trade via a trusted exchange. Exchanges benefit from access to the network's aggregated liquidity and can provide deeper order books and new trading pairs to their users.
+**Traders** use [OpenDEX-enabled exchanges](https://opendex.network/trade/exchanges) as gateways to trade on OpenDEX. These exchanges are usually non-custodial and abstract away from the technological complexity of the DEX and offer a superior UX by enabling trading via a simple website. **Exchanges** use `xud` to hedge user trades on OpenDEX in order to lock in profits. Liquidity on OpenDEX is provided by **market makers**, who use `xud` to take orders from external exchanges (e.g. Binance) and offer these on OpenDEX for a premium. Exchange Union provides SDK's for both, exchanges to automate hedging with `xud` and for market makers to connect to external exchanges and automate the arbitrage trades.
 
 ## Get Started
 
--> [Get started as **User**](User%20Guide.md), buying and selling cryptocurrency
+-> [Get started as **Trader**](User%20Guide.md), buying and selling cryptocurrency anonymously
 
--> [Get started as **Exchange**](Exchange%20Guide.md), tapping into the network's liquidity
+-> [Get started as **Market Maker**](Market%20Maker%20Guide.md), providing liquidity from external exchanges making a profit
+
+-> [Get started as **Exchange Operator**](), using a open-source white-label exchange platform to run your own exchange **(coming soon!)**
 
 -> [Get started as **Developer**](Developer%20Guide.md), contributing or building on top of `xud`
 
 ![Trading via XUD](/images/orderbook.png)
 
+## Incentives
+* Traders benefit from anonymous peer-to-peer trading without KYC and accounts.
+* Market makers make profits via arbitrage between external exchanges and OpenDEX.
+* Exchanges make profits via locking in trading fees by hedging trades on OpenDEX.
+
 ## Features
-* No central point or authority.
-* No middleman.
-* No KYC.
-* Supports individual traders *and* exchanges.
-* Market makers earn fees.
-* Direct, peer-to-peer trading.
+* Supports individual traders, market makers and *and* exchanges.
```

Comment:
Oxford comma here after "market makers" would be nice, to clearly separate individual traders from market makers.

Reply:
done.